Description/en-US.md

```diff
@@ -1,75 +1,67 @@
 # English (US)
+<!-- Since EN-US is supposed to be the "default" description, omit this header when pasting to distribution platforms -->
 
-A Fabric-based modpack for Minecraft that focuses on performance and graphics optimizations. 
+A Fabric-based Minecraft modpack focusing on performance and graphics enhancements.
 
-[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
+[<img src="https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg" alt="Current review" height="140"/> - a YouTube showcase by Rexus](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
 
 ### Performance
 
-Fabulously Optimized is usually faster than OptiFine due to the inclusion of [modern performance mods][1] like Sodium and Lithium.
+Fabulously Optimized is usually faster than OptiFine thanks to [modern performance mods][1] such as Sodium, Lithium and more!
 
-### OptiFine parity
+### Compatibility
```

Comment:
```suggestion
### Compatibility

```
just a spacing thing, doesn't actually change rendering on GH at least
Same for all the other headers that don't have newlines after

Reply:
Does this actually matter in rendering (anywhere)? I'd rather leave it without this spacing, so it's more readable.

Reply:
> Does this actually matter in rendering (anywhere)? I'd rather leave it without this spacing, so it's more readable.

Mostly just [my VS Code](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) throwing warnings via [MD022](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md022), which I like to think of as best practice.

Some markdown renderers need an extra newline to signify the start of a new line instead of just line wrapping, however I don't believe this applies to headings, I just tend to follow [markdownlint's rules](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md) when writing markdown.